test/test_helper.exs

```diff
@@ -6,5 +6,9 @@ defmodule TestHelper do
     def read(filename) do
       File.read!("test/fixtures/#{filename}")
     end
+
+    def dummy do
+      File.read!("test/fixtures/dummy.json")
```

Comment:
yes, but I think for most things it is a good idea still. For example, in my tests, we are testing lists. I still used the list(s) fixtures but dummied out the associations. In those associations, I would use a fixture that matched the association.